src/key_gen/message.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use super::{Commitment, Part};
+use crate::id::PublicId;
+use std::collections::BTreeSet;
+use std::fmt;
+
+/// Messages used for running BLS DKG.
+#[serde(bound = "")]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum DkgMessage<P: PublicId> {
```

Comment:
Agreed, no need for dkg to be everywhere.

Reply:
ok. will drop the `dkg` prefix

Reply:
@madadam : removing dkg from `DkgResult` will cause a confliction to the keyword of `Result`.
As this will be used outside the crate, shall it be kept?

Reply:
Then either keep the prefix or rename to something like `Outcome` or `Output`.